webapp/src/main/webapp/src/app/groups/import/group-import.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit, ElementRef, ViewChild } from "@angular/core";
+import { GroupImportService } from "./group-import.service";
+import { ImportResult } from "./import-result.model";
+import { FileUploader } from "ng2-file-upload";
+import { isNullOrUndefined } from "util";
+
+const URL = '/api/studentGroups/';
+
+@Component({
+  selector: 'admin',
+  templateUrl: 'group-import.component.html'
+})
+export class GroupImportComponent implements OnInit {
+
+  @ViewChild("fileDialog")
+  fileDialog: ElementRef;
+
+  studentGroupBatches: ImportResult[] = [];
+  public uploader: FileUploader;
+  public hasDropZoneOver: boolean;
+
+  constructor(private studentGroupService: GroupImportService) {
+  }
+
+  ngOnInit() {
+    this.uploader = new FileUploader({ url: URL });
+    this.uploader.setOptions({ autoUpload: true});
+
+    this.uploader.onCompleteItem = (item: any, response: any, status: any, headers: any) => {
+      console.log(item);
+      if (!isNullOrUndefined(response)) {
+
+        let importResult = this.studentGroupService.mapStudentGroupBatchFromApi(JSON.parse(response));
+        importResult.fileName = item.file.name;
```

Comment:
Grabbing the filename from the event.  This will need to be persisted for when we want to see this on the history page.